test/CMakeLists.txt

```diff
@@ -26,6 +26,10 @@ function(add_fmt_executable name)
   if (MINGW)
     target_link_libraries(${name} -static-libgcc -static-libstdc++)
   endif ()
+  if (CMAKE_CXX_COMPILER_ID MATCHES "GNU" AND
```

Comment:
Could you add a link to https://gcc.gnu.org/bugzilla/show_bug.cgi?id=88443 or whatever the most relevant gcc bug is?

Also is it fixed in gcc 11?

Reply:
Done.

There are no warnings in ```gcc version 11.1.1 20210721 [revision 076930b9690ac3564638636f6b13bbb6bc608aea] (SUSE Linux)```.